Rx.NET/Source/src/System.Reactive/System.Reactive.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0;net472;uap10.0.16299;net5.0;net5.0-windows10.0.19041</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;net472;uap10.0.16299</TargetFrameworks>
```

Comment:
Any idea why it supports 472 rather than 462 (as 462 was the first fully compliant netstandard2.0 version of the .NET Framework)?